caffe2/utils/math.h

```diff
@@ -207,6 +207,18 @@ void Broadcast(
     T* Y,
     Context* context);
 
+template <typename T, class Context>
+void Moments(
```

Comment:
What's moments? In maths, it usually comes with n-th order, right?

Reply:
It just comes from TensorFlow's moments which computes the mean and variance of a given tensor. Maybe we can consider a better name.
https://www.tensorflow.org/api_docs/python/tf/nn/moments

Reply:
I see. Mean and variance are first and second moments. Makes sense.